src/components/SpaceProposalVotesModal.vue

```diff
@@ -0,0 +1,238 @@
+      votes.value = [...to];
+    }
+  }
+);
+
+watch(
+  () => props.open,
+  to => {
+    if (!to) {
+      votes.value = [...props.votes];
+    }
+  }
+);
+</script>
+
+<template>
+  <BaseModal
+    :open="open"
+    max-height="500px"
+    max-width="700px"
+    @close="$emit('close')"
+  >
+    <template #header>
+      <div
+        class="flex flex-row content-center items-center justify-center gap-x-4 pl-4"
+      >
+        <h3>{{ $t('votes') }}</h3>
+        <BaseSearch
+          :model-value="voteQuery"
+          :placeholder="$t('searchPlaceholder')"
+          :autofocus="true"
+          class="flex-auto pr-2"
+        />
+      </div>
+    </template>
+    <div class="flex w-full flex-col">
+      <div
+        ref="votesScrollWrapper"
+        class="flex h-full flex-col overflow-auto"
+        :style="{ maxHeight: '450px' }"
+      >
+        <div
+          v-for="(vote, i) in sortedVotes"
+          :key="i"
+          :class="[
+            'flex items-center border-t px-4 py-[14px]',
+            { '!border-0': i === 0 }
+          ]"
+          :data-testid="`proposal-votes-list-item-${i}`"
+        >
+          <BaseUser
+            :key="vote.voter"
+            :profile="profiles[vote.voter]"
+            :address="vote.voter"
+            :space="space"
+            :proposal="proposal"
+            width-class="w-[110px] min-w-[110px] xs:w-[130px] xs:min-w-[130px] text-left"
+          />
+
+          <SpaceProposalVotesListItemChoice :proposal="proposal" :vote="vote" />
+          <div
+            class="flex w-[110px] min-w-[110px] items-center justify-end whitespace-nowrap text-right text-skin-link xs:w-[130px] xs:min-w-[130px]"
+          >
+            <span
+              v-tippy="{
+                content: vote.scores
+                  ?.map(
+                    (score, index) =>
+                      `${formatCompactNumber(score)} ${titles[index]}`
+                  )
+                  .join(' + ')
+              }"
+            >
+              {{
+                `${formatCompactNumber(vote.balance)} ${shorten(
+                  proposal.symbol || space.symbol,
+                  'symbol'
+                )}`
+              }}
+            </span>
+            <BaseButtonIcon @click="openReceiptModal(vote.ipfs)">
+              <BaseIcon name="signature" />
+            </BaseButtonIcon>
+            <BaseButtonIcon
+              v-if="vote.reason !== '' && props.proposal.privacy !== 'shutter'"
+              v-tippy="{
+                content: vote.reason
+              }"
+              class="cursor-default p-0"
+            >
+              <i-ho-annotation class="text-[16px]" />
+            </BaseButtonIcon>
+          </div>
+        </div>
+        <a
+          v-if="sortedVotes.length < voteCount"
+          class="block rounded-b-none border-t px-4 py-3 text-center md:rounded-b-md"
+          @click="loadMore(loadMoreVotes)"
+        >
+          <LoadingSpinner v-if="loadingMore" />
```

Comment:
The infinite scroll is buggy for me, sometimes when I scroll down I just see "See more" and sometimes I see the spinner.

Reply:
Yep I think at the end there will be only spinner

Reply:
Wdym?

Reply:
The infinite loading is super buggy for me https://www.loom.com/share/e3b88a02acb94edf94d50ec69240ae85

Reply:
Yep there was a throttle and small offset to bottom, fixed it.